projects/plugins/jetpack/changelog/gold-remove-thickbox

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: enhancement
+
+Remove thickbox integration for recurring payments.
```

Comment:
"Remove thickbox integration for Payments features, and thus remove jQuery dependency"